ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -227,7 +269,7 @@ open class ShimmerView: UIView {
 		let dark = UIColor.black.cgColor
 		let isRTL = effectiveUserInterfaceLayoutDirection == .rightToLeft
 
-		shimmeringLayer.colors = [dark, light, dark]
+		shimmeringLayer.colors = shimmerStyle == .concealing ? [light, dark, light] : [dark, light, dark]
```

Comment:
If the light and dark shimmering layer colors are moved to constants, this could be another property of the enum to avoid the ternary operator here?

Reply:
Good point, moved the layer colors to Shimmer Style

Reply:
Moved the dark color to the constant. Light color unfortunately cannot be part of constant because it depends on shimmerAlpha and is created dynamically. So unfortunately can't remove the ternary operator here.